frame/staking/src/pallet/impls.rs

```diff
@@ -629,54 +639,79 @@ impl<T: Config> Pallet<T> {
 
 	/// Get all of the voters that are eligible for the npos election.
 	///
-	/// This will use all on-chain nominators, and all the validators will inject a self vote.
+	/// `voter_count` imposes a cap on the number of voters returned; care should be taken to ensure
+	/// that it is accurate.
```

Comment:
It is a bit weird to ask for user to care without more detail.
```suggestion
	/// `maybe_max_len` can imposes a cap on the number of voters returned; First all the validator are included in no particular order, then the nominator in the order returned by the configured [`Config::SortedListProvider`].
```